README.md

```diff
@@ -54,8 +54,12 @@ cargo run -p trin-state|trin-history
 
 **Optional:** Custom data directory
 ```shell
-TRIN_DATA_PATH="/path"
+TRIN_DATA_PATH="/your_path"
 ```
+*Note, default data paths are:*\
+Linux/Unix - `$HOME/.local/share/trin` OR `~/.trin`
```

Comment:
what does the OR refer to? When does it use the former path and when the latter?

Reply:
I believe it just depends on the system. I'm running ubuntu, and my data was not stored in the first dir. @KolbyML had the second dir on their previously. The docs are here: https://docs.rs/directories/3.0.2/directories/struct.ProjectDirs.html#method.data_local_dir

Reply:
I don't think it will store files in ``~/.trin``. That is the directory I wanted to use, but other members wanted follow the XRC spec or something.